arch/arm64/boot/dts/qcom/sdm632-motorola-ocean.dts

```diff
@@ -457,6 +457,8 @@
 };
 
 &wcnss_iris {
+	compatible = "qcom,wcn3660b"; /* WCN3680B */
```

Comment:
Use `qcom,wcn3680` if your device has a WCN3680B

Reply:
Afaik moto ocean has 3660b it was find out by @ultra-azu

Reply:
https://github.com/ultra-azu/linux/commits/alikates/6.5/main